harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -776,6 +636,29 @@ def train_batch(
         """
         pass
 
+    def _train_batch(
```

Comment:
blocking: we should not be putting any logic as part of the PytorchTrial class, this should live as part of our own pytorch framework logic on TrialController.